src/algorithms/searching/linearSearch.js

```diff
@@ -0,0 +1,10 @@
+function linearSearch(list, itemToFind) {
+  for (let i = 0; i < list.length; i++) {
+    if (list[i] === itemToFind) {
+      return true;
```

Comment:
a better option is to return the index of the first occurrence of the element. in case the element is not located, then return null

Reply:
Fixed it!